packages/parany/parany.12.2.1/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/UnixJunkie/parany"
+bug-reports: "https://github.com/UnixJunkie/parany/issues"
+dev-repo: "git+https://github.com/UnixJunkie/parany.git"
+depends: [
+  "base-unix"
+  "cpu"
+  "dune" {>= "1.6.0"}
+  "ocaml" {>= "4.03.0"}
+]
```

Comment:
```suggestion
]
available: [ os != "windows" ]
```

Something like that might be appropriate if the package is not intended to be usable on windows. Maybe the same change should be applied to other versions of the same package?